ui/src/app/services/sign-in.service.ts

```diff
@@ -39,12 +40,13 @@ export class SignInService {
   // Expose "current user details" as an Observable
   public user: Observable<SignInDetails>;
   public currentAccessToken: string = null;
+  // TODO(calbach): Move into server config?
```

Comment:
@dmohs should I attempt this? Seems minimally we'll need to move this to `environment.ts`  to support multiple environments.